### DISCUSSION TITLE
UBOL Firefox

### DISCUSSION THREAD

**Discussion by Ethan-Swift** [uBlockOrigin/uBOL-home#205] [Q&A]
Hi, I'm confused about the intended install method for the Firefox uBOL version. I attempted to install via the new signed .xpi released a few days ago, but upon clicking the link and allowing the extension to download, Firefox blocks the install with a message saying that "Firefox has prevented this site from installing an unverified add-on." Additionally, when I try to force install via enterprise policy extension settings using the .xpi downloaded to my local system, Firefox gives me the policy error "Download failed - ERROR_SIGNEDSTATE_REQUIRED." I had assumed the file was already signed and that we did not need to manually sign the .xpi ourselves in order to install, but is that the intended solution here? Thanks!

**Comment by gorhill** [comment]
Somehow I fumbled and I didn't publish the signed package. It's fixed now.

**Reply by Ethan-Swift** [reply]
Yep works perfect now thank you!
